drivers/sensor/scd4x/scd4x.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022 Stephen Oliver
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */   
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SCD4X_SCD4X_H_
+#define ZEPHYR_DRIVERS_SENSOR_SCD4X_SCD4X_H_
+
+#include <zephyr/device.h>
+
+
+#define SCD4X_MAX_AMBIENT_PRESSURE UINT16_MAX
+
+#define SCD4X_CMD_POWER_DOWN 	0x36E0
```

Comment:
If you have these defined inside an enum I think that would make the `cmd` arg for functions like `scd4x_write_reg` to have a clear link to these command codes

same goes for the register addresses definitions, and functions that deal with registers